src/shared/util.js

```diff
@@ -282,15 +282,21 @@ export function looseEqual (a: any, b: any): boolean {
         return a.length === b.length && a.every((e, i) => {
           return looseEqual(e, b[i])
         })
-      } else if (!isArrayA && !isArrayB) {
-        const keysA = Object.keys(a)
-        const keysB = Object.keys(b)
-        return keysA.length === keysB.length && keysA.every(key => {
-          return looseEqual(a[key], b[key])
-        })
       } else {
-        /* istanbul ignore next */
-        return false
+        const isDateA = a instanceof Date
```

Comment:
hmm, why did you added it this way instead of another `else if` condition?

Reply:
I originally defined the variables at the top below the isArrayA, and isArrayB variables, but that was calling `instanceof` for everything that got passed in and added about 500 milliseconds to the test run. It's my first time perusing the code base. Not sure how much this method is used and if it would have a significant impact on anything else.

Also, I originally called instanceof directly inside the if, but tried to follow the code style of putting them in variables first.